### PR TITLE
Fixing some issues in configuring backend service Url in Dashboard

### PR DESCRIPTION
**Pull request by piraveena**

### CODE REVIEW

File: modules/jaggery-apps/portal/conf/site.json [hunk 1]
```diff
@@ -1,6 +1,11 @@
 {
     "server" : {
```
Comment (malithie) [inline modules/jaggery-apps/portal/conf/site.json]: I think we changed this to 'service'

**Comment by malithie** [comment]
Link this to the issue and mention the approach followed

**Comment by piraveena** [comment]
This is related to the issue [4434](https://github.com/wso2/product-is/issues/4434). 

When IS is fronted with LB, the dashboard works only if reverse proxy is configured in IS (reverse proxy can be configured by configuring hostname and proxy port in carbon.xml file and catalina-server.xml files respectively). 

With this fix,  dashboard can work without configuring carbon.xml and catalina-server.xml files. If reverse proxy is not configured in IS side, then we need to  configure host name and port in site.json files, SAMLIdpUrl , and sso-idp-config.xml

File: modules/jaggery-apps/portal/util/utility.jag [hunk 2]
```diff
@@ -23,18 +23,60 @@ function setServerURL() {
...
         var serverCoreUrl = '';
 
-        var serviceOrigin = jsonConf.server.serviceOrigin;
+        var mgtTransport = "https";
         var fidoAppId = jsonConf.fido.appId;
+        //retrieving site configuration for overiding ui urls
+        var proxyConfHost = jsonConf.proxy.proxyHost;
+        var proxyConfPort = jsonConf.proxy.proxyHTTPSPort;
+        var proxyOrigin = '';
+        var proxyContextPath = ''
```
Comment (omindu) [inline modules/jaggery-apps/portal/util/utility.jag]: missing `;`

**Pull request merged**